api/envoy/config/trace/v3/lightstep.proto

```diff
@@ -40,9 +42,14 @@ message LightstepConfig {
   // The cluster manager cluster that hosts the LightStep collectors.
   string collector_cluster = 1 [(validate.rules).string = {min_len: 1}];
 
-  // File containing the access token to the `LightStep
-  // <https://lightstep.com/>`_ API.
-  string access_token_file = 2 [(validate.rules).string = {min_len: 1}];
+  oneof access_token_setting {
```

Comment:
You can't upgrade to oneof, it breaks the backward compatibility in generated code (e.g. Golang), which isn't allowed in our API policy. We should deprecate `access_token_file` and use DataSource.

Reply:
OK, done.